src/ape/api/transactions.py

```diff
@@ -54,12 +54,6 @@ def total_transfer_value(self) -> int:
 
         return self.value + self.max_fee
 
-    @abstractmethod
-    def serialize_transaction(self) -> bytes:
```

Comment:
This method is called in at least two places other than blocks (providers and accounts).
Maybe move the logic to `EcosystemAPI` if you need it out of here?